internals/cli/cli.go

```diff
@@ -214,92 +208,57 @@ func Parser(cli *client.Client) *flags.Parser {
+			if strings.HasPrefix(specifier, "--") {
+				optionsHelp[specifier] = help
+			} else if utf8.RuneCountInString(specifier) == 2 && strings.HasPrefix(specifier, "-") {
+				optionsHelp[specifier] = help
+			} else if strings.HasPrefix(specifier, "<") && strings.HasSuffix(specifier, ">") {
+				positionalArgsHelp[specifier] = help
+			} else {
+				logger.Panicf("invalid help specifier: %#v %#v", c.Name, strings.HasPrefix(specifier, "-"))
 			}
 		}
 
-		args := cmd.Args()
-		if c.ArgumentsHelp != nil && len(args) != len(c.ArgumentsHelp) {
-			logger.Panicf("wrong number of argument descriptions for %s: expected %d, got %d", c.Name, len(args), len(c.ArgumentsHelp))
-		}
-		for _, arg := range args {
-			name, desc := arg.Name, ""
-			if c.ArgumentsHelp != nil {
-				name = c.ArgumentsHelp[name].Placeholder
-				desc = c.ArgumentsHelp[name].Help
-			}
-			lintArg(c.Name, name, desc, arg.Description)
-			name = fixupArg(name)
-			arg.Name = name
-			arg.Description = desc
-		}
-	}
-	// Add the debug command
-	debugCommand, err := parser.AddCommand("debug", shortDebugHelp, longDebugHelp, &cmdDebug{})
-	debugCommand.Hidden = true
-	if err != nil {
-		logger.Panicf("cannot add command %q: %v", "debug", err)
-	}
-	// Add all the sub-commands of the debug command
-	for _, c := range debugCommands {
-		obj := c.Builder()
-		if x, ok := obj.(clientSetter); ok {
-			x.setClient(cli)
-		}
-		cmd, err := debugCommand.AddCommand(c.Name, c.Summary, strings.TrimSpace(c.Description), obj)
-		if err != nil {
-			logger.Panicf("cannot add debug command %q: %v", c.Name, err)
-		}
+		hasAnyOptionHelp := len(optionsHelp) > 0
+		hasAnyPositionalHelp := len(positionalArgsHelp) > 0
+
+		// Check either all or none opts/positional argument descriptions are set
 		opts := cmd.Options()
-		if c.OptionsHelp != nil && len(opts) != len(c.OptionsHelp) {
-			logger.Panicf("wrong number of option descriptions for %s: expected %d, got %d", c.Name, len(opts), len(c.OptionsHelp))
+		if hasAnyOptionHelp && len(opts) != len(optionsHelp) {
+			logger.Panicf("wrong number of option descriptions for %s: expected %d, got %d", c.Name, len(opts), len(optionsHelp))
 		}
+		args := cmd.Args()
+		if hasAnyPositionalHelp && len(args) != len(positionalArgsHelp) {
+			logger.Panicf("wrong number of argument descriptions for %s: expected %d, got %d", c.Name, len(args), len(positionalArgsHelp))
+		}
+
 		for _, opt := range opts {
-			name := opt.LongName
-			if name == "" {
-				name = string(opt.ShortName)
-			}
-			desc, ok := c.OptionsHelp[name]
-			if !(c.OptionsHelp == nil || ok) {
-				logger.Panicf("%s missing description for %s", c.Name, name)
-			}
-			lintDesc(c.Name, name, desc, opt.Description)
-			if desc != "" {
-				opt.Description = desc
+			if description, ok := optionsHelp["--"+opt.LongName]; ok {
+				lintDesc(c.Name, opt.LongName, description, opt.Description)
+				opt.Description = description
+			} else if description, ok := optionsHelp["-"+string(opt.ShortName)]; ok {
```

Comment:
Why the string conversion here?

Reply:
`opt.ShortName` is a `rune`, not a `string`.